frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx

```diff
@@ -316,6 +347,12 @@ export function AddTeamMember() {
         handleCloseToast={handleCloseToast}
         severity="success"
       />
+      <SimpleToast
```

Comment:
please modify this to be generic i.e if the api is success, it should show success message and if api fails, it should show error message. please take reference from - https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/pages/ContactUs/ContactUs.jsx#L324-L328

Reply:
Ok I will update it with generic one.